aries-site/src/pages/design-tokens/versioning.mdx

```diff
@@ -0,0 +1,58 @@
+import { Box, Grid, Image } from 'grommet';
+import { Example } from '../../layouts';
+
+<Example
+  caption="Dashboard with ^0.9.0 (existing theme) applied on left and ^1.0.0 (updated theme) applied on right"
+  width="100%"
+>
+  <Grid columns={['flex', 'flex']} gap="medium" align="start">
+    <Box>
+      <Image src="/hpe-design-tokens-v0.png" fit="contain" />
+    </Box>
+    <Box>
+      <Image src="/hpe-design-tokens-v1.png" fit="contain" />
+    </Box>
+  </Grid>
+</Example>
+
+## Production-ready versions
+
+### ^1.0.0
+
+This release provides production-ready support with styles in alignment with the updated HPE theme.
+
+In the v2 Figma Component Library, this is the default theme.
+
+### ^0.9.0
+
+This release provides production-ready support with styles in alignment with the old HPE theme (grommet-theme-hpe v5).
+
+To ease adoption of design tokens, v0.9.0 allows teams with existing applications to adopt design tokens while maintaining styling that aligns with existing theming.
```

Comment:
Wondering if we can pull this messaging into an "info" notification following the "In the v2 Figma Component Library, this can be opted into by changing variables to use the "v0" modes."

```suggestion
```

Reply:
Done locally.

Reply:
Still seeing this come through in addition to the notification